development/docker-compose.yml

```diff
@@ -39,7 +49,10 @@ services:
       timeout: 5s
       start_period: 5s
       retries: 3
-      test: ["CMD", "nautobot-server", "health_check"]
+      test: 
+        - "bash"
+        - "-c"
```

Comment:
For my education, why do we need `bash -c` here instead of `CMD` as in the cases above?